kustomize/move2kube/README.md

```diff
@@ -0,0 +1,82 @@
+## Prerequisites
+* The manifests are generated without the `namespace` configuration
+* Manifest names reflect the `resources` section of [kustomization.yaml](./base/kustomization.yaml)
+* Apply the required configuration described in the Configuration section of [INSTALL.md](../../charts/workflows/charts/move2kube/INSTALL.md#configuration) 
+ 
+A brief explanation of the system architecture is provided in the related [document](./move2kube.md).
+
+## Kustomization options
+### Image update
+Run the following to update the default images to a custom configuration
+```
+cd base && kustomize edit set image serverless-workflow-move2kube=quay.io/orchestrator/serverless-workflow-move2kube:1234 && cd ..
+cd base && kustomize edit set image serverless-workflow-m2k-kfunc=quay.io/orchestrator/serverless-workflow-m2k-kfunc:1234 && cd ..
+```
+
+## Configure properties
+Edit configuration in [values.properties](./base/values.properties), [config.properties](./base/config.properties) and 
+[secret.properties](./base/secret.properties) to match your environment configuration.
+
+## Deploy to the cluster
+### Deploy the production environment
+This environment applies the generated manifests with minimal customizations to:
+* Use the latest application images
+* Force the `prod` profile for the SonataFlow instance
+* Deploy by default on the `sonataflow-infra` namespace
+* Mount the configurations defined in [Configure properties](#configure-properties) as environment variables
+
+The default namespace can be customized by running the following:
+```bash
+TARGET_NS=YOUR-NS
+```
+Then apply the changes with:
+```bash
+cd base && kustomize edit set namespace $TARGET_NS && cd ..
+```
+
+Once the configuration is set, apply the deployment to the configured namespace with:
+```bash
+kustomize build base | oc apply -f -
+```
+
+You can monitor the deployment status with:
+```bash
+oc get sonataflow m2k -n ${TARGET_NS} -owide
+```
+
+And finally view the logs with:
+```bash
+oc logs -f -n ${TARGET_NS} -l app=m2k
+```
```

Comment:
IMO, even if you reference the INSTALL.md file, you should copy/paste the steps when you update the move2kube URLs: provide the command to do it or at least the location of the file and the props to edit and then put the kustomize apply command again to apply those and then the command to restart the m2k pods to reflect the changes of the CM to the pods

Reply:
I'm not following you here: are you referring to the pre-installation steps or the fact that whenever you update the CM/Secret mounted to the Pod's `envFrom` the Pod is not restarted? (which is a different issue, and BTW tracked by https://github.com/apache/incubator-kie-kogito-serverless-operator/issues/298)
Pls clarify

Reply:
Updated the README to clarify that the `MOVE2KUBE_URL` and `MOVE2KUBE_API` properties must match the exposed Route, but I would not make it a 2 steps installation. On the contrary, I would be tempted to split the configuration in 2 to isolate the installation of `Move2Kube` that, BTW, may also be already installed in the cluster or anywhere else. But not today, of course 😉

Reply:
> I would be tempted to split the configuration in 2 to isolate the installation of Move2Kube

That would be fine for me too, like in `move2kube.md` too add/move some steps
But that is for later for sure